static/js/apps/nl_eval/model_match_box.tsx

```diff
@@ -62,7 +62,7 @@ function findKNearestEmbeddings(
   return result.slice(0, k);
 }
 
-export function SearchResult(props: SearchResultProps): JSX.Element {
+export function ModelMatchBox(props: ModelMatchBoxProps): JSX.Element {
```

Comment:
edit: maybe ModelScoreSection? ModelMatch makes me think we're matching things to the models

Reply:
changed